### PR TITLE
Feature/implement colorblend structure

### PR DESCRIPTION
**Pull request by m-herrera**

### CODE REVIEW

File: ext/tiovx/gsttiovx.c [hunk 1]
```diff
@@ -68,6 +68,7 @@
 #include "gsttiovxcolorconvert.h"
 #include "gsttiovxmultiscaler.h"
 #include "gsttiovxdlpreproc.h"
+#include "gsttiovxdlcolorblend.h"
```
Comment (rrcarlosrodriguez) [inline ext/tiovx/gsttiovx.c:71]: Alphabetical order

File: ext/tiovx/gsttiovx.c [hunk 2]
```diff
@@ -89,6 +90,9 @@ ti_ovx_init (GstPlugin * plugin)
 
   ret = gst_element_register (plugin, "tiovxdlpreproc", GST_RANK_NONE,
       GST_TIOVX_TYPE_DL_PRE_PROC);
+
+  ret = gst_element_register (plugin, "tiovxdlcolorblend", GST_RANK_NONE,
```
Comment (rrcarlosrodriguez) [inline ext/tiovx/gsttiovx.c:94]: Check if register fail and put and error message

File: ext/tiovx/gsttiovxdlcolorblend.c [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+{
+  GObjectClass *gobject_class = NULL;
+  GstElementClass *gstelement_class = NULL;
+  GstTIOVXMisoClass *gsttiovxmiso_class = NULL;
+
+  gobject_class = G_OBJECT_CLASS (klass);
+  gstelement_class = GST_ELEMENT_CLASS (klass);
+  gsttiovxmiso_class = GST_TIOVX_MISO_CLASS (klass);
+
+/* TODO Update element description*/
```
Comment (rrcarlosrodriguez) [inline ext/tiovx/gsttiovxdlcolorblend.c:176]: Remove this comment

File: ext/tiovx/gsttiovxdlcolorblend.h [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+ * GST_TIOVX_IS_DL_COLOR_BLEND:
+ * @ptr: pointer to check if it's a TIOVX color blend
+ * 
+ * Checks if a pointer is a TIOVX color blend
+ * 
+ * Returns: TRUE if @ptr is a TIOVX color blend
+ * 
+ */
+#define GST_TIOVX_TYPE_DL_COLOR_BLEND (gst_tiovx_dl_color_blend_get_type())
+G_DECLARE_FINAL_TYPE(GstTIOVXDLColorBlend, gst_tiovx_dl_color_blend, GST_TIOVX,
```
Comment (rrcarlosrodriguez) [inline ext/tiovx/gsttiovxdlcolorblend.h:83]: This declaration is not similar to the one used in multiscaler and colorconvert, why?

**Review by rrcarlosrodriguez — CHANGES_REQUESTED** [review]

File: ext/tiovx/gsttiovxdlcolorblend.c [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+#include "gsttiovxdlcolorblend.h"
+
+struct _GstTIOVXDLColorBlend
+{
+  GstTIOVXMiso element;
+};
+
+/* Formats definition */
+#define TIOVX_DL_COLOR_BLEND_SUPPORTED_FORMATS_SRC "ANY"
+#define TIOVX_DL_COLOR_BLEND_SUPPORTED_FORMATS_SINK "ANY"
```
Comment (emmadrigal) [inline ext/tiovx/gsttiovxdlcolorblend.c:77]: Is this correct?

File: ext/tiovx/meson.build [hunk 6]
```diff
@@ -5,13 +5,15 @@ gst_plugin_sources = [
...
   'gsttiovxdlpreproc.c',
+  'gsttiovxdlcolorblend.c',
 ]
 
 gst_plugin_headers = [
   'gsttiovxcolorconvert.h',
   'gsttiovxmultiscaler.h',
   'gsttiovxmultiscalerpad.h',
   'gsttiovxdlpreproc.h',
+  'gsttiovxdlcolorblend.h'
```
Comment (emmadrigal) [inline ext/tiovx/meson.build:16]: Alphabetical order

**Review by emmadrigal — CHANGES_REQUESTED** [review]

File: ext/tiovx/gsttiovxdlcolorblend.c (hunk 5 above)
Comment (m-herrera) [reply ext/tiovx/gsttiovxdlcolorblend.c:77]: It is not, but we don't have the supported formats yet, I will update them as soon as they are defined

Comment (emmadrigal) [reply ext/tiovx/gsttiovxdlcolorblend.c:77]: ok

**Pull request merged**